### ISSUE TITLE
oracle.simulate_shift() for multiple genes

### ISSUE THREAD

**Issue by TimelikeTachyon** [morris-lab/CellOracle#114]
Dear authors,

Fantastic package and documentation! The in silico TF perturbation result fits our human H9 model nicely. 
One quick question, I'm wondering if the simulation function, such as oracle.simulate_shift(), have any pontentials dealing with multiple genes simultaneously? It seems that multiple gene KO (especially multiple genes that from the same gene family) is kind of  "common" protocol in some experiment designs. Any replies will be much appreciated. Thanks!

Best,
Goingecho

**Comment by KenjiKamimoto-wustl122** [comment]
@TimelikeTachyon 
Thank you for trying celloracle and giving us feedback.

It is computationally possible to do multiple-TF perturbation simulation. Please enter multiple gene conditions in the dictionary and use the dictionary in the simulation function.
For example, If you want to try "Gata1" gene and "Gata2" gene in this [tutorial section,](https://morris-lab.github.io/CellOracle.documentation/notebooks/05_simulation/Gata1_KO_simulation_with_Paul_etal_2015_data.html#3.-In-silico-TF-perturbation-analysis) please run the simulation function.
`oracle.simulate_shift(perturb_condition={"Gata1": 0.0, "Gata2": 0.0})`

But we did not introduce such multiple gene knockout simulation mainly because of the following reasons.
1. In our paper, we wanted to simplify the analysis and experimental design to focus on the concept of CellOracle. 
2. In our paper, we needed large-scale validation using experimental gene perturbation data. It was relatively easy to get single gene perturbation experimental results. But, it was more difficult to get multiple gene knockout results. So we thought we could not do thorough validation on the multiple gene knockout results, and we decided not to do multiple gene knockout analysis in our paper.

In summary, we did not do multiple gene perturbation analysis so far but we agree with you that it is important to do multiple gene perturbation in experimental designs. This is one of the important future directions.

Best,
Kenji
